proto/batch_plan.proto

```diff
@@ -24,6 +24,12 @@ message RowSeqScanNode {
   common.Buffer vnode_bitmap = 4;
   // Whether the order on output columns should be preserved.
   bool ordered = 5;
+
+  message ChunkSize {
+    uint32 chunk_size = 1;
+  }
+  // If along with `batch_limit`, `chunk_size` will be set.
+  ChunkSize chunk_size = 6;
```

Comment:
Why not `optional uint32 chunk_size`?

Reply:
I remember long ago we forbid `optional` in proto3 for some reason, but it seems to work now. 😂 Both LGTM.